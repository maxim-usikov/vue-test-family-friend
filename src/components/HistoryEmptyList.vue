<template>
  <div :class="$style.Container">
    No data
  </div>
</template>

<script>
export default {
  name: 'HistoryEmptyList',
};
</script>

<style module>
.Container {
  background: rgba(234, 234, 234, 1);
  list-style: none;
  margin: 0;
  padding: 10px;
}
</style>
