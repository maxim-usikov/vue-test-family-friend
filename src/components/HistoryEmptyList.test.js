import { shallowMount } from '@vue/test-utils';

import HistoryEmptyList from './HistoryEmptyList.vue';

describe('HistoryEmptyList', () => {
  let $c;

  beforeEach(() => {
    $c = shallowMount(HistoryEmptyList);
  });

  it('is a Vue instance', () => {
    expect($c.isVueInstance()).toBeTruthy();
  });

  it('name is `HistoryEmptyList`', () => {
    expect($c.name()).toBe('HistoryEmptyList');
  });

  it('renders correctly', () => {
    expect($c.element).toMatchSnapshot();
  });
});
