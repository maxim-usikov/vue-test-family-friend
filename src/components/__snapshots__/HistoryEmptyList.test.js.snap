// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryEmptyList renders correctly 1`] = `
<div
  class="Container"
>
  
  No data

</div>
`;
